tutorials/javaFxPart3.md

```diff
@@ -7,15 +7,20 @@
 
 Picking up from where we left off last tutorial, we have successfully achieved the desired layout. Now let’s make the application respond to user input.
 
-Rather than to do everything in one try, let’s iterate and build up towards our final goal.
+JavaFX has an _event-driven architecture style_. As such, we programmatically define _handler_ methods to execute as a response to certain _events_. "Events" refers to the [Event class](https://docs.oracle.com/javase/8/javafx/api/javafx/event/Event.html). Two of the most commonly used subclasses are [ActionEvent](https://docs.oracle.com/javase/8/javafx/api/javafx/event/Event.html) (such as a button fired or a key frame finished) and [InputEvent](https://docs.oracle.com/javase/8/javafx/api/javafx/event/Event.html) (such as a mouse click).
```

Comment:
Should we be citing Java 8 pages here?

Reply:
Maybe these are the links that should be used instead, can't seem to find JavaFX docs in [oracle Java17 docs](https://docs.oracle.com/en/java/javase/17/docs/api/index.html), but I think these docs should be fine as well, considering the JavaFX version we use is also from openjfx.

https://openjfx.io/javadoc/17/javafx.base/javafx/event/Event.html
https://openjfx.io/javadoc/17/javafx.base/javafx/event/ActionEvent.html
https://openjfx.io/javadoc/17/javafx.graphics/javafx/scene/input/InputEvent.html